draft-ietf-masque-h3-datagram.md

```diff
@@ -230,9 +229,21 @@ Capsule Data:
 
 : Data whose semantics depends on the Capsule Type.
 
+Unless otherwise specified, all Capsule Types are defined as opaque to
+intermediaries. Intermediaries MUST forward all received opaque CAPSULE frames
+in their unmodified entirety. Intermediaries MUST NOT send any opaque CAPSULE
+frames other than the ones it is forwarding. All Capsule Types defined in this
+document are opaque, with the exception of the DATAGRAM Capsule, see
+{{datagram-capsule}}. Definitions of new Capsule Types MAY specify that the
+newly introduced type is transparent. Intermediaries MUST treat unknown Capsule
+Types as opaque.
+
+Intermediaries respect the order of opaque CAPSULE frames: if an intermediary
+receives two opaque CAPSULE frames in a given order, it MUST forward them in
+the same order.
+
 Endpoints which receive a Capsule with an unknown Capsule Type MUST silently
-drop that Capsule. Intermediaries MUST forward Capsules, even if they do not
-know the Capsule Type or cannot parse the Capsule Data.
+drop that Capsule.
```

Comment:
This para seems a bit like a fragment, maybe you could just append the clause to the end of the sentence 2 paras up? I.e. "Intermediaries MUST treat unknown Capsule Types as opaque and MUST silently drop the Capsule."

Reply:
For intermediaries it's MUST forward, not MUST drop. Otherwise new capsules won't work without modifying intermediaries.

Reply:
oh sorry I misread things